docs/medhelm.md

```diff
@@ -1,243 +1,298 @@
-    - Recording Research Processes
-    - Ensuring Clinical Research Quality
-    - Managing Research Enrollment
-
-- **Administration and Workflow**
-    - Scheduling Resources and Staff
-    - Overseeing Financial Activities
-    - Organizing Workflow Processes
-    - Care Coordination and Planning
-
-This categorization ensures that evaluations reflect the **complexity, diversity, and stakes** of medical AI applications—from assisting clinicians in making critical decisions to safely interacting with patients.
+> **Who it’s for:** Data scientists at health systems benchmarking LLMs for medical use cases.
+>
+> **What you’ll do here:** Install MedHELM, run a small evaluation locally, understand access levels, view leaderboards, and learn how to contribute new scenarios/models.
+>
+> **Time required:** \~10 minutes for the Quickstart.
+>
+> **Requirements:** [Conda](https://www.anaconda.com/docs/getting-started/getting-started)/Python 3.10 (~30 min), [Google Cloud CLI](https://cloud.google.com/sdk/docs/install) (~30 min), and GPU access (exact GPU requierements depend on the model used and context length).
 
-## References
 
-- [Stanford HAI Article](https://hai.stanford.edu/news/holistic-evaluation-of-large-language-models-for-medical-applications)
-- [MedHELM Website](https://crfm.stanford.edu/helm/medhelm/latest/)
-- [Publication](https://arxiv.org/abs/2505.23802)
+MedHELM extends the HELM framework to evaluate **large language models (LLMs) in medical applications**, focusing on realistic tasks, safety, and reproducibility.
 
-## Installation
 
-First, follow the [installation instructions](installation.md) to install the base HELM framework. For improved stability, we recommend using Python 3.10.
+## Quickstart (10 minutes)
 
-To install MedHELM-specific dependencies:
+**Goal:** Install MedHELM, run a 10-instance evaluation on a public scenario, and open a local leaderboard.
 
-```sh
-pip install "crfm-helm[summarization]" && pip install "crfm-helm[medhelm]"
+**Prerequisites:** Python 3.10, ability to create a virtual environment via [conda](https://www.anaconda.com/docs/getting-started/getting-started).
+
+### 1) Install (Python 3.10 recommended)
+
+Run the following commands to create and activate a new python virtual environment:
+
+```bash
+# Create and activate a clean environment
+conda create -n crfm-helm python=3.10 pip
+conda activate crfm-helm
+pip install -U setuptools
+```
+
+Run the following command to install HELM:
+```
+pip install crfm-helm
 ```
 
-## Getting Started
+Run the following commands to install the necessary MedHELM extensions:
 
-The following is an example of evaluating [Qwen2.5-7B-Instruct](https://huggingface.co/Qwen/Qwen2.5-7B-Instruct) on the [MedCalc-Bench scenario](https://github.com/stanford-crfm/helm/blob/main/src/helm/benchmark/scenarios/medcalc_bench_scenario.py) using 10 instances.
+```
+# Install MedHELM extras
+pip install "crfm-helm[summarization]" && pip install "crfm-helm[medhelm]"
```

Comment:
you should install everything in one step `pip install "crfm-helm[summarization,medhelm]"`so that the dependency resolution happens all at once.